static/js/pages/images/ImageName.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { FC } from "react";
+import { queryKeys } from "util/queryKeys";
+import { fetchImage } from "api/images";
+import { useQuery } from "@tanstack/react-query";
+
+interface Props {
+  id: string;
+}
+
+const ImageName: FC<Props> = ({ id }) => {
+  const { data: image } = useQuery({
+    queryKey: [queryKeys.images, id],
+    queryFn: () => fetchImage(id),
+  });
+
+  return <>{image ? image.properties.description : id}</>;
```

Comment:
Shouldn't we use the `project` parameter for the query here?
Also, since we're falling back on the `id` (the fingerprint), which is a very long chunk of text, I'd still apply the `truncate` and `title` to be safe.

Reply:
True about the truncate / fallback.

The images are independent of projects, so it should be fine with the network call as it is.